halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/local/git/LocalGitDeckService.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2017 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.halyard.deploy.spinnaker.v1.service.local.git;
+
+import com.netflix.spinnaker.halyard.config.model.v1.node.DeploymentConfiguration;
+import com.netflix.spinnaker.halyard.deploy.deployment.v1.DeploymentDetails;
+import com.netflix.spinnaker.halyard.deploy.services.v1.ArtifactService;
+import com.netflix.spinnaker.halyard.deploy.spinnaker.v1.SpinnakerRuntimeSettings;
+import com.netflix.spinnaker.halyard.deploy.spinnaker.v1.profile.Profile;
+import com.netflix.spinnaker.halyard.deploy.spinnaker.v1.service.DeckService;
+import com.netflix.spinnaker.halyard.deploy.spinnaker.v1.service.ServiceSettings;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@EqualsAndHashCode(callSuper = true)
+@Data
+@Component
+public class LocalGitDeckService extends DeckService implements LocalGitService<DeckService.Deck> {
+  String deskSettingsPath = "settings.js";
+  String deckPath = "~/.spinnaker/" + deskSettingsPath;
+
+  String startCommand = "export SETTINGS_PATH=" + deckPath + "\n"
+      + "yarn > /dev/null\n"
+      + "yarn start";
+
+  @Autowired
+  String gitRoot;
+
+  @Autowired
+  ArtifactService artifactService;
+
+  @Override
+  public ServiceSettings buildServiceSettings(DeploymentConfiguration deploymentConfiguration) {
```

Comment:
Does this match exactly what's in the super class? if so, we can skip this

Reply:
buildServiceSettings is not implemented anywhere above this.